bedrock/firefox/templates/firefox/browsers/chromebook.html

```diff
@@ -27,8 +27,7 @@
 
 {% block content %}
   {% call split(
-    image_url='img/firefox/browsers/quantum/browser.jpg',
-    include_highres_image=True,
+    image=resp_img('img/firefox/browsers/quantum/browser.jpg', srcset={ 'img/firefox/browsers/quantum/browser.jpg': '2x' }, optional_attributes={ 'class': 'mzp-c-split-media-asset'}),
```

Comment:
Nit: some of these lines get pretty long, suggest indenting for better readability?

Reply:
```suggestion
    image=resp_img('img/firefox/browsers/quantum/browser.jpg', srcset={ 'img/firefox/browsers/quantum/browser-high-res.jpg': '2x' }, optional_attributes={ 'class': 'mzp-c-split-media-asset'}),
```